CSharpMath.Tests/PreTypesetting/MathListBuilderTest.cs

```diff
@@ -938,19 +941,19 @@ public class MathListBuilderTest {
       Assert.Equal(MathAtomType.LargeOperator, op.AtomType);
       Assert.False(op.Limits);
       var latex = MathListBuilder.MathListToString(list);
-      Assert.Equal(@"\sum \nolimits ", latex);
+      Assert.Equal(RemoveWhiteSpacesFromString(latex), input);

```

Comment:
The first argument of `Assert` is the expected value, the second argument is the actual value. Because of that, these two arguments should be inverted.